tests/ui/features/licenseexport_spdx.feature

```diff
@@ -51,39 +51,35 @@ Scenario: Verify the headers on SPDX SBOM License reference CSV file
 	When User selects a package with Single license information
-	Then "name" column should match "name" from SBOM
-	And "namespace" column should match "documentNamespace" from SBOM
-	And "package reference" column should match "packages.externalRefs.referenceLocator" of "packages.externalRefs.referenceType" type purl from SBOM
-	And "license expression" column should match "packages.licenseDeclared" from SBOM
-	And The columns "group", "version", "license id", "license name", "alternate package reference" should be empty
+	Then "SBOM name" column should match "name" from SBOM
+	And "SBOM id" column should match "namespace" from SBOM
+	And "package name" column should match "packages.name" from SBOM
+	And "package group" column should match "packages.group" from SBOM
+	And "package version" column should match "packages.versionInfo" from SBOM
+	And "package purl" column should match "packages.externalRefs.referenceLocator" of "packages.externalRefs.referenceType" purl from SBOM
+	And "license" column should match "packages.licenseDeclared" from SBOM
+	And "package cpe" column should be empty
 
 Scenario: Verify the license information for a package with single license with alternate package reference referenceLocator
 	Given User is on SBOM license information file
-	When User selects a package with Single license information
-	Then "name" column should match "name" from SBOM
-	And "namespace" column should match "documentNamespace" from SBOM
-	And "package reference" column should match "packages.externalRefs.referenceLocator" of "packages.externalRefs.referenceType" purl from SBOM
-	And "license expression" column should match "packages.licenseDeclared" from SBOM
-	And "alternate package reference" column should match "packages.externalRefs.referenceLocator" of "packages.externalRefs.referenceType" type cpe from SBOM json
-	And The columns "group", "version", "license id", "license name" should be empty
-
-Scenario: Verify the license information for a package with multiple licenses with alternate package reference referenceLocator
-	Given User is on SBOM license information file
-	When User selects a package with Single license information
-	Then "name" column should match "name" from SBOM
-	And "namespace" column should match "documentNamespace" from SBOM
-	And "package reference" column should match "packages.externalRefs.referenceLocator" of "packages.externalRefs.referenceType" purl from SBOM
-	And "license expression" column should match the whole value of "packages.licenseDeclared" from SBOM in a single row
-	And "alternate package reference" column should match "packages.externalRefs.referenceLocator" of "packages.externalRefs.referenceType" type cpe from SBOM json
-	And The columns "group", "version", "license id", "license name" should be empty
+	When User selects a package with Single license with cpe information
+	Then "SBOM name" column should match "name" from SBOM
+	And "SBOM id" column should match "namespace" from SBOM
```

Comment:
I see this is matching `documentNamespace` not a `namespace` property from SPDX SBOM JSON ... i do not have there just 'namespace' named prop.

Reply:
e.g.
```
{
 "spdxVersion": "SPDX-2.3",
 "dataLicense": "CC0-1.0",
 "SPDXID": "SPDXRef-DOCUMENT",
 "name": "registry.k8s.io/coredns/coredns:v1.9.3",
 "documentNamespace": "https://anchore.com/syft/image/registry.k8s.io/coredns/coredns-v1.9.3-feed6dea-b0bc-483a-be72-0c3229d8d3e6",
 "creationInfo": {
  "licenseListVersion": "3.19",
  "creators": [
   "Organization: Anchore, Inc",
   "Tool: syft-0.65.0"
  ],
  "created": "2023-05-17T17:07:54Z",
  "comment": ""
 },
 "packages": [
  {
   "name": "cloud.google.com/go/compute",
   "SPDXID": "SPDXRef-Package-go-module-cloud.google.com-go-compute-36357a787dfd18c0",
   "versionInfo": "v1.6.1",
```

Reply:
exported in licenses.csv:
```
"SBOM name"	"SBOM id"	"package name"	"package group"	"package version"	"package purl"	"package cpe"	"license"
"registry.k8s.io/coredns/coredns:v1.9.3"	"https://anchore.com/syft/image/registry.k8s.io/coredns/coredns-v1.9.3-feed6dea-b0bc-483a-be72-0c3229d8d3e6"	"cloud.google.com/go/compute"	""	"v1.6.1"	"pkg:golang/cloud.google.com/go/compute@v1.6.1"	""	"NONE"
```

Reply:
Nice catch, it is documentNamespace